app/javascript/helpers/custom-click-handler.js

```diff
@@ -0,0 +1,10 @@
+/** Function to handle onclick events generated from remote_functions.
+ * Note: This was introduced to remove the usage of 'eval' feature.
```

Comment:
@jeffibm I don't think we need this Note.  Also, naive scanners will still see the word eval here and flag us anyway 🤦